src/editors/data/redux/video/selectors.js

```diff
@@ -1,16 +1,20 @@
-// import { createSelector } from 'reselect';
+import { createSelector } from 'reselect';
 
 import { keyStore } from '../../../utils';
+import { videoTranscriptLanguages } from '../../constants/video';
 
 import { initialState } from './reducer';
-// import * as module from './selectors';
+import * as module from './selectors';
+import * as AppSelectors from '../app/selectors';
+import { downloadVideoTranscriptURL } from '../../services/cms/urls';
 
 const stateKeys = keyStore(initialState);
 
 export const video = (state) => state.video;
 
 export const simpleSelectors = [
   stateKeys.videoSource,
+  stateKeys.videoId,
```

Comment:
Is this specifically edx video id?

Reply:
yes, it is needed for the transcript API calls